packages/utils/src/labelValue.ts

```diff
@@ -10,7 +10,16 @@ import { ReactElement } from 'react';
  * @param [fallback] - One of 3 values, `undefined` (the default), `false` or an empty string
  * @returns - `fallback` if `hideLabel` is true, otherwise `label`
  */
-export function labelValue(label?: string | ReactElement, hideLabel?: boolean, fallback?: ''): undefined | string;
```

Comment:
@heath-freenome the main issue why what we tried during weekly meeting was not working. was missing `default` in export statement

Reply:
Would just changing it to the following work without the need for the types?
```
export default function labelValue(label?: string, hideLabel?: boolean, fallback?: ''): undefined | string;
```

Reply:
the generic version is a bit more specific it also fixes cases where `ReactElement` might be not assignable to `string` property (some of removed suppressions are about that case)

it technically would be equivalent to this:

```typescript
export default function labelValue(label?: string, hideLabel?: boolean, fallback?: ''): undefined | string;
export default function labelValue(label?: string, hideLabel?: boolean, fallback?: false): undefined | false | string;
export default function labelValue(label?: ReactElement, hideLabel?: boolean, fallback?: ''): undefined | ReactElement;
export default function labelValue(
  label?: ReactElement,
  hideLabel?: boolean,
  fallback?: false
): undefined | false | ReactElement;
export default function labelValue(
  label?: string | ReactElement,
  hideLabel?: boolean,
  fallback?: false | ''
): undefined | false | string | ReactElement {
  return hideLabel ? fallback : label;
}
```

do not really have preferences, here… generic however is a bit smaller

Reply:
Let's avoid the generic so that we don't have to update the Documentation.

Reply:
updated